vpc/vpc.tf

```diff
@@ -0,0 +1,116 @@
+# VPC for Open Oakland
+# Setup our aws provider
+provider "aws" {
+  region = "${var.vpc_region}"
+}
```

Comment:
Chatted with @meipang , we agree this should live in the terraform consumer rather than the module.